authentication/src/main/java/com/ecommercepractice/authentication/service/MobileService.java

```diff
@@ -0,0 +1,22 @@
+package com.ecommercepractice.authentication.service;
+
+import com.ecommercepractice.authentication.dao.MobileDao;
+import com.ecommercepractice.authentication.model.MobileInfoModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MobileService {
+    @Autowired
+    MobileDao mobileDao;
+
+    /**
+     * Store mobile info onto repository
+     * @param mobileInfo
+     * @return
+     */
```

Comment:
please explain to me the reason to return the same object that is used as a param